[Zulip->*] Messages not sent over bridge

**Describe the bug**
Messages are not read from Zulip streams into IRC channel.

**To Reproduce**
1. Set up matterbridge on docker with the instruction on the wiki or use the binary locally.
2. Make a Zulip<->IRC bridge.
3. Run the container/program.

**Expected behavior**
Messages are transmitted to the IRC channel from the Zulip stream.
**NOTE:** IRC messages reach the Zulip stream properly.
~~Also, adding, I had to run it on Docker because using the binary directly was not working for me (unsure why), if the instructions were more clearer I could clarify that as well.~~
Tested without a container and same issue persists. Note that for this, I used the current release (and not development).

**Debug logs**
```
user@user-MacBook-Pro:matterbridge % docker run -ti -v ~/matterbridge/matterbridge.toml:/matterbridge.toml 42wim/matterbridge -debug
[0000]  INFO main:         Enabling debug logging.
[0000]  INFO main:         Running version 1.16.6-dev 6b017b22
[0000]  INFO main:         WARNING: THIS IS A DEVELOPMENT VERSION. Things may break.
[0000]  INFO router:       Parsing gateway zulipirc
[0000]  INFO router:       Starting bridge: irc.freenode 
[0000]  INFO irc:          Connecting irc.freenode.net:6697
[0000] DEBUG irc:          "CLIENT_INIT irc.freenode.net:6697"
[0001] DEBUG irc:          ":card.freenode.net NOTICE * :*** Looking up your hostname..."
[0001] DEBUG irc:          ":card.freenode.net NOTICE * :*** Checking Ident"
[0001] DEBUG irc:          ":card.freenode.net NOTICE * :*** Couldn't look up your hostname"
[0010] DEBUG irc:          ":card.freenode.net NOTICE * :*** No Ident response"
[0010] DEBUG irc:          ":card.freenode.net CAP * LS :account-notify away-notify cap-notify chghost extended-join identify-msg multi-prefix sasl tls"
[0010] DEBUG irc:          ":card.freenode.net CAP zulipbridgebot ACK :cap-notify chghost multi-prefix account-notify extended-join sasl away-notify "
[0017] DEBUG irc:          "AUTHENTICATE +"
[0018] DEBUG irc:          ":card.freenode.net 900 zulipbridgebot zulipbridgebot!zulipbridg@xxx.xx.xx.xxx zulipbridgebot :You are now logged in as zulipbridgebot."
[0018] DEBUG irc:          ":card.freenode.net 903 zulipbridgebot :SASL authentication successful"
[0018] DEBUG irc:          ":card.freenode.net 001 zulipbridgebot :Welcome to the freenode Internet Relay Chat Network zulipbridgebot"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          Registering callbacks
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0018] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0019] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0019] DEBUG irc:          "CLIENT_GENERAL_UPDATED"
[0020] DEBUG irc:          "CLIENT_CONNECTED irc.freenode.net:6697"
[0024] DEBUG irc:          ":zulipbridgebot MODE zulipbridgebot +Zi"
[0024]  INFO irc:          Connection succeeded
[0024]  INFO irc:          irc.freenode: joining #wikimedia-test (ID: #wikimedia-testirc.freenode)
[0024]  INFO router:       Starting bridge: zulip.bot 
[0024] DEBUG irc:          handle girc.Event{Source:(*girc.Source)(0xc000527b90), Tags:girc.Tags(nil), Timestamp:time.Time{wall:0xbf95ad53d4d50867, ext:24734955744, loc:(*time.Location)(0x55feb8ff64e0)}, Command:"JOIN", Params:[]string{"#wikimedia-test", "zulipbridgebot", "zulipbridgebot"}, Sensitive:false, Echo:false}
[0025]  INFO zulip:        Connection succeeded
[0025]  INFO zulip:        zulip.bot: joining bot/topic:IRC (ID: bot/topic:IRCzulip.bot)
[0025]  INFO main:         Gateway(s) started succesfully. Now relaying messages
[0025] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0026] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0026] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0026] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0026] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0027] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0028] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0028] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0029] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0029] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0029] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
[0030] DEBUG zulip:        receiving error: &errors.errorString{s:"Error was unknown"}
```

**Environment (please complete the following information):**
 - OS: Mac OS X Catalina 10.15.2
 - Matterbridge version: master (1.16.6-dev 6b017b22)

**Additional context**
```
[irc]
	[irc.freenode]
	Server="irc.freenode.net:6697"
	Nick="zulipbridgebot"
	NickServNick="zulipbridgebot"
	NickServPassword=""
	UseTLS=true
	UseSASL=true
	SkipTLSVerify=false
	RemoteNickFormat="[{PROTOCOL}] <{NICK}> "

[zulip]
	[zulip.bot]
	Token=""
	Login=""
	Server=""
	RemoteNickFormat="[{PROTOCOL}] <{NICK}>"

[[gateway]]
name="zulipirc"
enable=true
    [[gateway.inout]]
    account="irc.freenode"
    channel="#wikimedia-test"
    [[gateway.inout]]
    account="zulip.bot"
    channel=""

```

The gozulipbot author, @ifo had kindly fixed the `getEvents` bug (see https://github.com/ifo/gozulipbot/issues/2). I tested the latest master of https://github.com/ifo/gozulipbot after patching these errors
```
bridge/zulip/zulip.go:62:18: b.bot.UpdateMessage undefined (type *gozulipbot.Bot has no field or method UpdateMessage)
bridge/zulip/zulip.go:78:18: b.bot.UpdateMessage undefined (type *gozulipbot.Bot has no field or method UpdateMessage)
bridge/zulip/zulip.go:90:23: b.bot.GetRawStreams undefined (type *gozulipbot.Bot has no field or method GetRawStreams)
bridge/zulip/zulip.go:110:8: undefined: gozulipbot.NoJSONError
bridge/zulip/zulip.go:113:8: undefined: gozulipbot.BadEventQueueError
bridge/zulip/zulip.go:141:29: m.StreamID undefined (type gozulipbot.EventMessage has no field or method StreamID)
```
using https://github.com/42wim/matterbridge/tree/master/vendor/github.com/matterbridge/gozulipbot as a reference.

Messages from IRC are being correctly relayed to Zulip, but still not the other way around.

Maybe @42wim can test the bugfixes better than me.

--- a/bridge/zulip/zulip.go
+++ b/bridge/zulip/zulip.go
@@ -146,8 +146,8 @@ func (b *Bzulip) handleQueue() error {
 			b.Log.Debugf("<= Sending message from %s on %s to gateway", rmsg.Username, b.Account)
 			b.Log.Debugf("<= Message is %#v", rmsg)
 			b.Remote <- rmsg
-			b.q.LastEventID = m.ID
 		}
+
 		time.Sleep(time.Second * 3)
 	}
 }

